### PR TITLE
HoC 2023 - Update How-to guides • Companies

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update Companies How-to guide: https://hourofcode.com/how-to/companies
- Page is a draft: can be seen locally at http://localhost.hourofcode.com:3000/us/en/how-to/companies-draft
- Will update each How-to page as a draft and set everything live in a separate PR
- Cleaned up a few things from the [Educators PR](https://github.com/code-dot-org/code-dot-org/pull/54384)

**Related PRs:**
- https://github.com/code-dot-org/code-dot-org/pull/54384
- https://github.com/code-dot-org/code-dot-org/pull/54557

**Jira ticket:** [ACQ-1009](https://codedotorg.atlassian.net/browse/ACQ-1009)

----

localhost hourofcode com_3000_us_how-to_companies-draft

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/how-to/index-draft.haml [hunk 1]
```diff
@@ -8,7 +8,7 @@ theme: how_to_guide
 = view :"how-to/how_to_banner", nav_audience: "guide_nav_educators"
 
 -# Intro section
-= view :"how-to/how_to_intro", intro_header: hoc_s(:hoc_how_to_educators_intro_heading), intro_desc: hoc_s(:hoc_how_to_educators_intro_desc)
+= view :"how-to/how_to_intro", intro_header: hoc_s(:hoc_how_to_educators_intro_heading), intro_desc: hoc_s(:hoc_how_to_educators_intro_desc), intro_desc_02: ""
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/how-to/index-draft.haml:11]: All of the other how-to guides have two paragraphs in this section so I added this empty string on the Educators (/index) one here so I could include an `intro_desc_02` variable on the `how_to_intro` view ([see here](https://github.com/code-dot-org/code-dot-org/pull/54548/files#diff-79b5466f5d14ba7c7c19f6bdd505436aa12506ae2efa994dec5b0c361010e2db))

File: pegasus/sites.v3/hourofcode.com/views/how-to/faq/faq_educators.haml
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/views/how-to/faq/faq_educators.haml:1]: Forgot to add the `resolve_url` method on these links on the Educators page FAQ.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -8440,6 +8444,10 @@
   hoc_how_to_educators_intro_heading: "How to plan your Hour of Code"
   hoc_how_to_educators_intro_desc: "The Hour of Code is a one-hour introduction to computer science, using fun tutorials to show that anybody can learn the basics. This year, we're taking Hour of Code to new heights by offering coding opportunities that encompass both artificial intelligence (AI) and non-AI components. Whether you or your students are experienced coders or you all are setting out on your very first computer science expedition, this Hour of Code promises an extraordinary voyage."
 
+  hoc_how_to_companies_intro_heading: "How your company can engage with the Hour of Code"
+  hoc_how_to_companies_intro_desc_01: "TThe Hour of Code is a one-hour introduction to computer science, using fun tutorials to show that anybody can learn the basics. This year, we're taking Hour of Code to new heights by offering coding opportunities that encompass both artificial intelligence (AI) and non-AI components. "
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: I think typo at the start with "TThe"

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -8494,6 +8520,13 @@
   hoc_how_to_step_educators_celebrate_desc: "After your event, print certificates of completion and congratulate your students. Share photos and videos of your Hour of Code event and show off your students' achievements using #HourOfCode and @codeorg."
   hoc_how_to_step_educators_beyond_desc: "Computer science doesn't have to end with the Hour of Code! While 90% of parents want their child to learn computer science, most schools still don't teach it. To help, our curriculum is web-based and free to use, forever. Learn how to bring CS to your school today."
 
+  hoc_how_to_step_companies_inspire_desc_01: "Share this video at an All Staff meeting or in an email to all employees to show how companies like yours make a huge impact on the global movement to bring computer science education to all students in all schools through the Hour of Code."
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Should this read "...how companies like yours **can** make a huge..."?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 4]
```diff
@@ -8494,6 +8520,13 @@
...
+  hoc_how_to_step_companies_encourage_desc_01: "One of the most fulfilling ways to participate in the Hour of Code is to volunteer with a local classroom, either in-person or virtually. Best of all, you dont have to be an engineer or someone with a technical background in order to volunteer. You can still provide a meaningful experience to students by sharing your own career experience and how computer science or technology has impacted your role. Direct your employees to the volunteer guide to learn more."
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Nit: "don't" instead of "dont"

**Review by TurnerRiley — APPROVED** [review]
Amazing as always! Just a few text nits

**Pull request merged**